Added latest version of manuscript example files as baseline for prop…

matthiaskoenig opened the pull request:
…osed changes

matthiaskoenig committed:
Added latest version of manuscript example files as baseline for proposed changes

matthiaskoenig committed:
SBML example updated, fixed path entries, updated README.

matthiaskoenig committed:
CellML example updated

matthiaskoenig committed:
Descriptions added to RDF

matthiaskoenig committed:
removing IDE files

matthiaskoenig committed:
descriptions added to metadata

matthiaskoenig committed:
Fixing the creator information

matthiaskoenig committed:
fixed incorrect identifiers namespaces for anntoations

maxneal reviewed (changes requested):
Hi Matthias. Just a couple things I noticed - it looks like the RDF content in the CellML model file got flattened into one line.
And apparently I used my gmail address when creating the CellML example when I should have used my CIDResearch address. So could you change instances of maxneal@gmail.com to max.neal@cidresearch.org in the CellML example metadata.rdf file? (Hat tip to Alan Garny for noticing this.)

Also, Martin Scharm noticed that in the metadata.rdf files for these example archives, I am listed as the creator of the cellml model and the sbml model. I certainly did not create them, just downloaded them and put them in the archive. So maybe I shouldn't be listed as the creator?

matthiaskoenig committed:
Full executable example with external annotations

matthiaskoenig committed:
Fixed creator issues

matthiaskoenig commented:
I updated the creator issues, i.e. 
- [x] fixed email Max
- [x] removed Max as creator form model files

In addition I now added the full SBML example with
- all annotations of SBML externalized as XML and Turtle RDF serialization
- SED-ML simulation experiment with results reproducing Figure from paper

This should be good to merge. Please have a look @maxneal

maxneal commented:
I think the RDF content in the CellML file is still flattened. If it helps, I think you could just revert this file back to the original one I sent out with the last version of the paper.

Otherwise things look good to merge in.
